justega247/favorite-things
# #167453252: User can view all categories

#### What does this PR do?
- This PR enables Users of the application to be able to view the categories that have been created in the application and subsequently be able to view new ones they create also
#### Description of Task to be completed?
- write the code to implement the functionality
#### How should this be manually tested?
- On your local version of this project
- Startup the backend server using **python manage.py runserver**
- Navigate into the client directory and start-up webpack using **npm start**
- Navigate to **http://localhost:3000/categories** on your browser
- You should see all the categories in the application and any new one you create
#### What are the relevant pivotal tracker stories?
[#167453252](https://www.pivotaltracker.com/story/show/167453252)
